build_tools/ci/cpu_comparison/matmul_template/matmul_trunci_scaling_MxK_KxN.mlir

```diff
@@ -0,0 +1,27 @@
+// input ${M}x${K}x${TYPE1}
+// input ${K}x${N}x${TYPE1}
+
+func.func @matmul_trunci(%arg0: tensor<${M}x${K}x${TYPE1}>, %arg1: tensor<${K}x${N}x${TYPE1}>) -> tensor<${M}x${N}x${TYPE1}>
```

Comment:
Needs a comment about what this template does

Seems like its 

C = ((A @ B) * cst_mul / cst_div) 

which a bit strange maybe, should the `*` be an `+`  (related to TYPE3 suggested by Jorn)

Function name misleading?

Reply:
No, it should be a multiplication + division to mimic a float scale to scale from i32 range to i8 range.

Reply:
Why are you trying to mimic that? (Documentation please Abhishek :-))

If the goal here is to get into a safe range where you don't overflow end-to-end test with large K with just a trunci, you could quite easily do that by choosing the initial values of A and B appropriately